content/applications/sales/crm/acquire_leads/lead_mining.rst

```diff
@@ -1,72 +1,89 @@
+- :guilabel:`Industries`: filter leads based on the specific industry they work in
+- :guilabel:`Sales Team`: choose which Sales Team the leads will be assigned to
+- :guilabel:`Salesperson`: choose which person(s) on the Sales Team the leads will be assigned to
+- :guilabel:`Default Tags`: choose which tags are applied directly to the leads once found
 
-When choosing to target Companied and their contacts you can choose the contacts you are getting based on Role or Seniority. 
-When getting contact information make sure to be aware of the latest EU regulation, get more information about General Data Protection Regulation on
-`Odoo GDPR <http://odoo.com/gdpr>`__, 
+.. note::
+   Make sure to be aware of the latest EU regulations when receiving contact information. Get more
+   information about the General Data Protection Regulation on `Odoo GDPR <http://odoo.com/gdpr>`__.
 
+Pricing
+=======
 
-You can decide to filter the leads you are getting depending on the size (number of employees) of the companies.
-You can pick the countries your leads are coming from. It is possible to pick multiple countries.
-You can pick the industries your leads are coming from. It is possible to pick multiple industries.
+Lead mining is an In-App Purchase feature and each generated lead costs one credit.
 
-You can choose the salesperson and the Salesteam the lead will be assigned to. 
-You can also add tags to track your generated leads.
+Choosing to generate :guilabel:`Companies and their Contacts` will cost one additional credit for
+each contact generated.
 
-.. tip::
-    The generated leads will have the name of the company.
+.. note::
+   See here for full pricing information: `Lead Generation by Odoo IAP <https://iap.odoo.com/iap/
+   in-app-services/167?>`__.
 
-Pricing
-==================
-This is an In-App Purchase feature, each generated lead will cost you one credit.
-If you choose to get contact information each contact will also cost us one additional credit.
-Here is the pricing for this `feature <https://iap.odoo.com/iap/in-app-services/167?>`__, 
-To buy credits you can either go to :menuselection:`CRM --> Configuration --> Settings --> Buy
-Credits`; or go to :menuselection:`Settings --> In-App Purchases --> View my Services`.
-
-.. image:: lead_mining/LM6.png
-   :align: center
+To buy credits, navigate to :menuselection:`CRM --> Configuration --> Settings`, and under the
+:guilabel:`Lead Mining` section, click on :guilabel:`Buy Credits`.
 
-.. image:: lead_mining/LM7.png
+.. image:: lead_mining/buy-lead-mining-credits-setting.png
    :align: center
+   :alt: Buy credits from lead mining settings.
 
+Credits may also be purchased by navigating to the :guilabel:`Settings` app. Under the
+:guilabel:`Odoo IAP` section, click on :guilabel:`View My Services`.
 
-.. tip::
-    The blue box will automatically tell you how many credits are going to be consumed.
+.. image:: lead_mining/buy-lead-mining-credits-iap.png
```

Comment:
Please replace the current image with the attached image, and change the image name to `lead_mining/view-my-services-setting.png`
![view-my-services-setting](https://user-images.githubusercontent.com/101904966/193158867-d3edc01c-f72d-4d1d-ad06-fd72e2444169.png)